x-pack/plugins/endpoint/public/applications/endpoint/types.ts

```diff
@@ -85,9 +90,23 @@ export interface EndpointAppLocation {
 }
 
 export type AlertListData = AlertResultList;
-export type AlertListState = Immutable<AlertResultList> & {
+
+export interface AlertListState {
```

Comment:
Defining a type for alert list state as opposed to making it equal to the type of the data the server returns.